app/models/account.rb

```diff
@@ -1,6 +1,8 @@
 class Account < ApplicationRecord
   include Syncable
 
+  validates :family_id, presence: true
```

Comment:
When validating associations, you need to pass `validates :family, presence: true` so that not only does the validation ensure the presence of a foreign key, but also that the associated record exists in the database.